### ISSUE TITLE
Verify the cluster that you're trying to work with is up

### ISSUE THREAD

**Issue by jorgemoralespou** [redhat-developer/ocdev#82]
As the tool relies on "oc" command, it can happen that a developer uses "ocdev" with a context selected in .kube/config of a cluster not accesible. A check should be made, or this error should be properly captured and reported.

```
$ ocdev application list
unable to list applications: unable to get projects: command: [/usr/local/bin/oc get project -o custom-columns=NAME:.metadata.name] failed to run:
Unable to connect to the server: dial tcp 192.168.64.40:8443: i/o timeout
: exit status 1
```

**Comment by containscafeine** [comment]
note to self: we can fix this by trying connecting to server from `oc whoami --show-server` and proceed with the result.

**Comment by kadel** [comment]
https://github.com/redhat-developer/ocdev/issues/125

**Comment by containscafeine** [comment]
Closed in #99
